contracts/src/v0.8/feeds/DualAggregator.sol

```diff
@@ -816,25 +816,24 @@ contract DualAggregator is OCR2Abstract, OwnerIsCreator, AggregatorV2V3Interface
         _payTransmitter(s_hotVars, report_.juelsPerFeeCoin, uint32(initialGas), msg.sender);
```

Comment:
It's possible the existing report is older than the current secondary round id.

Reply:
If an older report arrives it'll be `epochAndRound < s_hotVars.latestEpochAndRound` and revert with a `StaleReport()` error.

Reply:
I think you corrected it with lines 813-816